infra/modules/providers/azure/keyvault-cert/output.tf

```diff
@@ -0,0 +1,17 @@
+output "cert_name" {
+  value = "${var.key_vault_cert_name}"
+}
+
+output "public_cert" {
+  value     = "${lookup(data.external.public_cert.result, "cer")}"
+  sensitive = true
```

Comment:
Is this needed since its the public cert?

Reply:
We'll need the public cert for ssl termination.